File: src/codes/Users_Manual/device_1.cs
~using AUTD3Sharp;
~using AUTD3Sharp.Link;
~using AUTD3Sharp.Utils;
~using var autd = Controller.Builder([new AUTD3(Vector3.Zero)]).Open(Nop.Builder());
~{
  var tr = autd.Geometry[0][0];
  ~}
foreach (var tr in autd.Geometry[0])
{
  // do something
}

File: src/codes/Users_Manual/device_1.rs
# use autd3::prelude::*;
# 
# #[tokio::main]
# async fn main() -> Result<(), Box<dyn std::error::Error>> {
# let mut autd = Controller::builder([AUTD3::new(Vector3::zeros())])
#    .open(autd3::link::Nop::builder()).await?;
let tr = &autd.geometry[0][0];
for tr in &autd.geometry[0] {
    // do something
}
# Ok(())
# }